demo/index.html

```diff
@@ -3,16 +3,25 @@
         <meta charset="utf-8">
         <title>bcviz demo</title>
         <link rel="stylesheet" type="text/css" href="/src/css/bcviz.css">
+        <style>
+          fieldset { 
+            border-radius: 15px 15px 15px 15px; 
+            overflow: hidden; 
+            font-size: 14px; 
+          } 
+        </style>
         <script data-main="main" src="/external/requirejs/require.js"></script>
     </head>
     <body>
-        <div id="instructions">
-            <p>To switch to another set of graphs, use the &quot;.&quot; key to move forward and the &quot;,&quot; key to move backwards.</p>
```

Comment:
That's not the purpose of a demonstration program.

If such a program is needed, then tell me the requirements and use cases, and I can produce one.